.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,5 +32,5 @@
 
 ---     
  
->You can join our slack channel **wg-manifests** [here](https://www.kubeflow.org/docs/about/community/). This link also contains our meeting schedule.
+>You can join our CNCF Slack Channel **kubeflow-platform** [here](https://www.kubeflow.org/docs/about/community/). This link also contains our meeting schedule.
```

Comment:
please use [#kubeflow-platfrom](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2) to link directly to the channel and also keep the link to https://www.kubeflow.org/docs/about/community/ as well. Please make it the same in the issue template and PR template in all places.